jte/src/test/java/gg/jte/TemplateEngineTest.java

```diff
@@ -543,6 +543,132 @@ void tagWithVarArgs4() {
         thenOutputIs("test.key with Hello World ");
     }
 
+    @Test
+    void templateCall() {
+        dummyCodeResolver.givenCode("module/components/card.jte", "@param java.lang.String firstParam\n" +
+                "@param int secondParam\n" +
+                "One: ${firstParam}, two: ${secondParam}");
+        givenTemplate("@module.components.card(model.hello, model.x), That was an arbitrary template call!");
+        thenOutputIs("One: Hello, two: 42, That was an arbitrary template call!");
+    }
+
+    @Test
+    void templateCall_noConflictWithKeywords_for() {
+        dummyCodeResolver.givenCode("format.jte", "@param java.lang.String firstParam\n" +
+                "@param int secondParam\n" +
+                "One: ${firstParam}, two: ${secondParam}");
+        givenTemplate("@format(model.hello, model.x), That was an arbitrary template call!");
+        thenOutputIs("One: Hello, two: 42, That was an arbitrary template call!");
+    }
+
+    @Test
+    void templateCall_noConflictWithKeywords_param() {
+        dummyCodeResolver.givenCode("parameter.jte", "@param java.lang.String firstParam\n" +
+                "@param int secondParam\n" +
+                "One: ${firstParam}, two: ${secondParam}");
+        givenTemplate("@parameter(model.hello, model.x), That was an arbitrary template call!");
+        thenOutputIs("One: Hello, two: 42, That was an arbitrary template call!");
+    }
+
+    @Test
+    void templateCall_noConflictWithKeywords_import() {
+        dummyCodeResolver.givenCode("importer.jte", "@param java.lang.String firstParam\n" +
+                "@param int secondParam\n" +
+                "One: ${firstParam}, two: ${secondParam}");
+        givenTemplate("@importer(model.hello, model.x), That was an arbitrary template call!");
```

Comment:
I think we shouldn't allow arbitrary words right after `@`, as that limits extending the library with new keywords without BC breaks. Maybe something like `@include.importer`?

Reply:
Good point. The reason for arbitrary words is mainly to keep `@tag` and `@layout` working without special handling.

Reply:
As we discussed today, I've changed the keyword to `@template`. Thanks for the suggestions, I really like where this is going!